### ISSUE TITLE
(Feature request) Allow Styling of mini date selector

### ISSUE THREAD

**Issue by cwilliams5** [TriliumNext/Trilium#2637]
### Describe feature

Allow theme to style mini date selector, the way I can the launcher bar one. 

![image](https://user-images.githubusercontent.com/10100743/153699573-613d6ed4-5d4f-4f97-8894-b863f49ec01a.png)

### Additional Information

Unless I'm missing something, this is some sort of native Chromium/Electron control that can't be styled. I've tried all sorts of CSS wizardry and I can't style this thing. Only documentation I can find:
https://developers.google.com/web/updates/2012/08/Quick-FAQs-on-input-type-date-in-Google-Chrome#how_do_i_change_the_appearance_of_the_date_picker

That said, I know this would require building a whole new widget, so it may be low/no priority, but hey, throwing the request out there.

**Comment by eliandoran** [comment]
Indeed, the effort of integrating a custom widget here outweighs the benefits, unfortunately.
We might reconsider it at some point, if we ever renew our widgets.
